### PR TITLE
#164869458 View specific question for a specific meetup

### PR DESCRIPTION
**Pull request by a-braham**
#### What does this PR do?
- Has view endpoint for viewing a specific question
#### Description of Task to be completed?
- `GET  api/meetups/{meetup_id}/questions/{question_id}/question`
#### How should this be manually tested?
After cloning the repo
- cd into questioner, checkout to branch `bg-view-specific-question-164869458`
- run tests `python api/manage.py test`

#### Relevant PT stories
[#164869458](https://www.pivotaltracker.com/n/projects/2315729/stories/164869458)
#### Screenshots
Screenshot from 2019-03-26 11-21-07

### CODE REVIEW

**Comment by Amoswachira** [comment]
@a-braham don't forget to  resolve the merge conflicts

**Review by Amoswachira — COMMENTED** [review]
@a-braham don't forget to resolve the merge conflicts

**Comment by Ogutu-Brian** [comment]
Hello @a-braham ,
Would you add screenshot and relevant pivotal tracker stories to the PR

**Review by IssaIan — DISMISSED** [review]
But remember to add necessary screenshots as suggested. You could also reduce the number of return statements within a function next time. Otherwise great work.

**Review by Bakley — CHANGES_REQUESTED** [review]
Kindly update your documentation before merging
ab

**Review by Bakley — DISMISSED** [review]

**Comment by Bakley** [comment]
Hey, @a-braham I tested your PR and it meets my expectation. The test were passing and it was fetching the right meetup. The error message response was also good enough. Great work on this. Kindly add now the screenshots to the PR for easy visibility.

File: api/questions/urls.py [hunk 1]
```diff
@@ -16,4 +16,6 @@
          QuestionEditViews.as_view(), name='delete_question'),
     path('<str:m_id>/questions/<str:id>/',
          QuestionEditViews.as_view(), name='edit_question'),
+    path('<str:m_id>/questions/<str:id>/question',
```
Comment (davenjeru) [inline api/questions/urls.py]: This `/question` at the end is unnecessary @a-braham. The URL should be `GET /api/meetups/{meetupId}/questions/{questionId}`

File: api/config/swagger.py [hunk 2]
```diff
@@ -509,6 +509,25 @@ def schema_view_swagger(request):
                     ],
                     description='Fetching questions on a specific meetup'
                 ),
+                'View a question': coreapi.Link(
+                    url='/api/meetups/{meetupId}/questions/{questionId}/question',
```
Comment (davenjeru) [inline api/config/swagger.py]: This `/question` at the end is unnecessary @a-braham. The URL should be `GET /api/meetups/{meetupId}/questions/{questionId}`

**Review by davenjeru — CHANGES_REQUESTED** [review]

File: api/config/swagger.py (hunk 2 above)
Comment (IssaIan) [inline api/config/swagger.py]: Also change this url in the swagger documentation to what @davenjeru adviced.

**Review by IssaIan — CHANGES_REQUESTED** [review]

**Review by IssaIan — APPROVED** [review]

File: api/config/swagger.py [hunk 3]
```diff
@@ -705,4 +724,4 @@ def schema_view_swagger(request):
 
         }
     )
-    return response.Response(schema)
+    return response.Response(schema)
```
Comment (Philipotieno) [inline api/config/swagger.py]: @a-braham Could use autopep8  to make sure a new line is added at the end of the file and everything is to standard

**Review by Ogutu-Brian — APPROVED** [review]

**Review by Philipotieno — CHANGES_REQUESTED** [review]

**Review by Philipotieno — APPROVED** [review]

**Review by davenjeru — APPROVED** [review]

**Pull request merged**